src/formio-complete.js

```diff
@@ -1,3 +1,4 @@
+window.CKEDITOR_BASEPATH = 'node_modules/ckeditor/';
```

Comment:
When ```require``` is ran, that is done with browserify which creates a distribution. ```node_modules``` is never included with the distribution application. The CKEditor would just be included manually.